src/test/java/fi/helsinki/moodi/test/AbstractMoodiIntegrationTest.java

```diff
@@ -62,15 +62,16 @@
 public abstract class AbstractMoodiIntegrationTest {
     private static final String EMPTY_LIST_RESPONSE = "[]";
     private static final String EXPECTED_COURSE_ID = MoodleCourseBuilder.MOODLE_COURSE_ID_PREFIX + 102374742;
-    protected static final String ERROR_RESPONSE = "{\"exception\":\"webservice_access_exception\",\"errorcode\":\"accessexception\",\"message\":\"P\\u00e4\\u00e4syn hallinnan poikkeus\"}";
+    protected static final String ERROR_RESPONSE =
+        "{\"exception\":\"webservice_access_exception\",\"errorcode\":\"accessexception\",\"message\":\"P\\u00e4\\u00e4syn hallinnan poikkeus\"}";
     protected static final String EMPTY_OK_RESPONSE = "";
     protected static final String NULL_OK_RESPONSE = "null";
     protected static final String NULL_DATA_RESPONSE = "{\"data\": null}";
 
     protected static final int APPROVED_ENROLLMENT_STATUS_CODE = 3;
     protected static final int NON_APPROVED_ENROLLMENT_STATUS_CODE = 10;
 
-    private static final ObjectMapper testObjectMapper = new ObjectMapper();
+    private static final ObjectMapper TEST_OBJECT_MAPPER = new ObjectMapper();
```

Comment:
Koska tässä on kyseessä potentiaalisesti mutatoituva objektireferenssi olisi parempi nimetä tämä camelCasella (koska kyseessä ei ole vakio). Ks esim. https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case (pätee `Logger`in lisäksi muihinkin objektityyppeihin).

Checkstyleen ei taida saada konffattua poikkeusta ei-vakio static final-objekteille (https://github.com/checkstyle/checkstyle/issues/23), vaan täytyy lisätä regex-poikkeus muuttujan nimen mukaan (ks. unihow:n survicen checkstyle-konffi, missä `log(ger)`ille on konffattu poikkeus). Voisi tehdä vastaavan poikkeuksen tähän.

(Moodissa on muuten nyt näköjään monessa kohdassa `log`/`logger`in sijaan käytetty uppercasella nimettyä `LOGGER`ia, mikä antaa ymmärtää että kyseesä on vakio, vaikka logger-instanssia voisi mutatoida esim. log levelin asetuksella -- voisi  vaihtaa noi `LOGGER`it -> `logger`eiksi)